beacon-chain/db/kv/slasher.go

```diff
@@ -16,11 +16,10 @@ import (
 // epoch we have recorded the validator attested for.
 func (s *Store) LatestEpochAttestedForValidators(
```

Comment:
This function is refactored to become more idiomatic and easier to use for the caller